packages/@okta/vuepress-theme-default/components/Code.vue

```diff
@@ -5,9 +5,9 @@
     <!-- End Sidebar -->
 
     <!-- Begin Content -->
-    <div class="PageContent-main language-content">
+    <div class="PageContent-main language-content NoPaddingTop">
       <EditLink />
-      <Breadcrumb></Breadcrumb>
+      <Breadcrumb wrapStyle="width: 80%"></Breadcrumb>
```

Comment:
Isn't there a better option than embedding style directly?

Reply:
I need the style directly on the .breadcrumb.Wrap, but only in this instance.   It's not the best, but will go away with the Prose redesign